### PR TITLE
Akretion migration10/l10n br account

### PR DESCRIPTION
**Pull request by renatonlima**
replace #492

Migração para a versão 10.0 do módulo l10n_br_account

### CODE REVIEW

File: l10n_br_account/views/res_company_view.xml [hunk 1]
```diff
@@ -21,21 +21,21 @@
                                 <separator colspan="4" string="CNAE Secundários"/>
                                 <field colspan="4" nolabel="1" name="cnae_secondary_ids"/>
                             </page>
-                            <page string="Regine Normal" attrs="{'invisible': [('fiscal_type', '!=', '3')]}">
+                            <page string="Regine Normal" name="regine_normal" attrs="{'invisible': [('fiscal_type', '!=', '3')]}">
```
Comment (sadamo) [inline l10n_br_account/views/res_company_view.xml:24]: Typo. Should be "Regime Normal".

File: l10n_br_account/models/account_invoice.py [hunk 2]
```diff
@@ -119,22 +139,57 @@ def finalize_invoice_move_lines(self, move_lines):
...
+        if not kwargs.get('fiscal_category_id'):
+            return result
+
+        company = self.env['res.company'].browse(kwargs.get('company_id'))
+
+        fcategory = self.env['l10n_br_account.fiscal.category'].browse(
+            kwargs.get('fiscal_category_id'))
+        result['value']['journal_id'] = fcategory.property_journal.id
+        if not result['value'].get('journal_id', False):
+            raise except_orm(
```
Comment (sadamo) [inline l10n_br_account/models/account_invoice.py:186]: except_orm not imported. Btw, the convention continues to be "from odoo.exceptions import Warning as UserError"?

File: l10n_br_account/account_fiscal_position_workflow.xml [hunk 3]
```diff
@@ -1,79 +0,0 @@
...
-        <record id="fiscal_position_approved_unapproved_transition" model="workflow.transition">
-            <field name="act_from" ref="fiscal_position_approved_activity"/>
-            <field name="act_to" ref="fiscal_position_unapproved_activity"/>
-            <field name="signal">fiscal_category_unapproved</field>
-        </record>
-
-        <record id="fiscal_position_draft_approved_transition" model="workflow.transition">
-            <field name="act_from" ref="fiscal_position_draft_activity"/>
-            <field name="act_to" ref="fiscal_position_approved_activity"/>
-            <field name="signal">fiscal_category_approved</field>
```
Comment (sadamo) [inline l10n_br_account/account_fiscal_position_workflow.xml:63]: Without the workflow, we can't aprove the Fiscal Category

**Review by sadamo — CHANGES_REQUESTED** [review]
the Fiscal Position Rule mapping is not working properly.

File: l10n_br_account/models/account_invoice.py (hunk 2 above)
Comment (danimaribeiro) [reply l10n_br_account/models/account_invoice.py:186]: "from odoo.exceptions import UserError"

**Review by mileo — APPROVED** [review]
Considerada as considerações do @sadamo e @danimaribeiro esta ok!

**Comment by mileo** [comment]
FYC:

Fiz o merge desta branch em https://github.com/odoo-brazil/l10n-brazil-wip/tree/develop e estarei trabalhando em uma feature ref aos lançamentos contábeis da invoice.

**Comment by mileo** [comment]
Please cherry pick this commits:

https://github.com/OCA/l10n-brazil/pull/515/commits/0250c7db2aedaf3f9399e073f170c7f5c42f2ff5
https://github.com/OCA/l10n-brazil/pull/515/commits/ce76b1566c68dd429b9c438ce11cb3db7b0f3a0f
https://github.com/OCA/l10n-brazil/pull/515/commits/43018c14611ed67d572cb3759868614168fb1391

**Comment by mileo** [comment]
Replaced by: https://github.com/OCA/l10n-brazil/pull/516

**Comment by mileo** [comment]
Como este PR já esta parado faz muito tempo vou fecha-lo. Por favor caso a haja interesse favor reabrir o mesmo.

**Pull request closed without merging**